src/components/Icons/Icon.tsx

```diff
@@ -75,13 +76,13 @@ export default function Icon(props: React.PropsWithChildren<IconProps>) {
         children,
     } = props;
 
-    let icon_base_class = "icon";
+    const icon_base_class = "icon";
```

Comment:
The change form underscore to camelcase was done in other files, why not this one?

Reply:
Oh because I put `/* eslint-disable camelcase */` at the top of this file to deal with the fact that a lot of the svg filenames were in snake case.  I wish I could just disable eslint for that block ...

Reply:
Ah, makes sense. Maybe just update this variable name but leave the SVG filenames in snake case, since this variable is camelcase in other files.